README.md

```diff
@@ -121,6 +121,111 @@ connection
+|:-----------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|
+| [`bigint`][psql-bigint-ref]                    | [**`Long`**][java-long-ref], [`Boolean`][java-boolean-ref], [`Byte`][java-byte-ref], [`Short`][java-short-ref], [`Integer`][java-integer-ref] |
+| [`bit`][psql-bit-ref]                          | Not yet supported.|
+| [`bit varying`][psql-bit-ref]                  | Not yet supported.|
+| [`boolean or bool`][psql-boolean-ref]          | [`Boolean`][java-boolean-ref]|
+| [`box`][psql-box-ref]                          | Not yet supported.|
+| [`bytea`][psql-bytea-ref]                      | Not yet supported.|
+| [`character`][psql-character-ref]              | [`String`][java-string-ref]|
+| [`character varying`][psql-character-ref]      | [`String[]`][java-string-ref]|
+| [`cidr`] [psql-cidr-ref]                       | Not yet supported.|
+| [`circle`][psql-circle-ref]                    | Not yet supported.|
+| [`date`][psql-date-ref]                        | [`LocalDate`][java-ld-ref]|
+| [`double precision`][psql-floating-point-ref]  | [**`Double`**][java-double-ref], [`Float`][java-float-ref]|
+| [`inet`][psql-inet-ref]                        | [**`InetAddress`**][java-inet-ref]|
+| [`integer`][psql-integer-ref]                  | [**`Integer`**][java-integer-ref], [`Boolean`][java-boolean-ref], [`Byte`][java-byte-ref], [`Short`][java-short-ref], [`Long`][java-long-ref]|
+| [`interval`][psql-interval-ref]                | Not yet supported.|
+| [`json`][psql-json-ref]                        | Not yet supported.|
+| [`line`][psql-line-ref]                        | Not yet supported.|
+| [`lseg`][psql-lseq-ref]                        | Not yet supported.|
+| [`macaddr`][psql-macaddr-ref]                  | Not yet supported.|
+| [`macaddr8`][psql-macaddr8-ref]                | Not yet supported.|
+| [`money`][psql-money-ref]                      | Not yet supported.|
+| [`numeric`][psql-bignumeric-ref]               | [`BigDecimal`][java-bigdecimal-ref]|
+| [`path`][psql-path-ref]                        | Not yet supported.|
+| [`pg_lsn`][psql-pg_lsn-ref]                    | Not yet supported.|
+| [`point`][psql-point-ref]                      | Not yet supported.|
+| [`polygon`][psql-polygon-ref]                  | Not yet supported.|
+| [`real`][psql-real-ref]                        | [**`Float`**][java-float-ref], [`Double`][java-double-ref]|
+| [`smallint`][psql-smallint-ref]                | [**`Short`**][java-short-ref], [`Boolean`][java-boolean-ref], [`Byte`][java-byte-ref], [`Integer`][java-integer-ref], [`Long`][java-long-ref]|
+| [`smallserial`][psql-smallserial-ref]          | Not yet supported.|
```

Comment:
You might want to recheck on `smallserial` and `serial` (see [`PostgresTestSupport`](https://github.com/spring-projects/spring-data-r2dbc/blob/29f1edce0e42dfe669bf1f71ce5aa06a5cdf4a06/src/test/java/org/springframework/data/r2dbc/testing/PostgresTestSupport.java#L33))

Reply:
I missed those, will update with following 
| [`smallserial`][psql-smallserial-ref]          | [**`Integer`**][java-integer-ref], [`Boolean`][java-boolean-ref], [`Byte`][java-byte-ref], [`Short`][java-short-ref], [`Long`][java-long-ref]|
| [`serial`][psql-serial-ref]                    | [**`Long`**][java-long-ref], [`Boolean`][java-boolean-ref], [`Byte`][java-byte-ref], [`Short`][java-short-ref], [`Integer`][java-integer-ref]|
|